spec/features/support/users/editing_a_user.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Editing a user" do
+  before do
+    given_i_am_authenticated(user: admin)
+    and_a_user_exists
+    when_i_visit_the_user_edit_page
+  end
+
+  scenario "successfully changing details" do
+    when_i_fill_in_correct_details
```

Comment:
i'd reword this to begin with `and` so it flows with the steps in the before block